desktop/src/main/java/bisq/desktop/bisq.css

```diff
@@ -165,6 +165,9 @@ bg color of non edit textFields: fafafa
     -fx-selection-bar-non-focused: -fx-selection-bar;
 
     -fx-font-size: 13;
+}
+
+.root:dir(ltr){
```

Comment:
Why was this necessary?

Reply:
for some reason, setting `-fx-font-family` in the `.root` class was breaking the Bi-directional system in Javafx and thus texts with mixed language characters were messed up. I limited it to left-to-right direction only.